browser/src/docdispatcher.ts

```diff
@@ -227,6 +227,19 @@ class Dispatcher {
 			app.map.insertComment();
 		};
 
+		this.actionsMap['showcommentsnavigator'] = function (data?: any) {
+			if (
+				!document
+					.getElementById('navigation-sidebar')
```

Comment:
We should later invent some better method to check for navigator activation,
this one could be true even if we already sent the commands (very quick clicking, slow network).

example:
`app.map.navigator.showCommentsNavigator(commentId: number)`

then in the `Control.NavigatorPanel.ts` we can store/know current state and send commands if needed
https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.NavigatorPanel.ts

Reply:
could you apply this change?